### DISCUSSION TITLE
Launching URLs in different browser instances rather than as tabs

### DISCUSSION THREAD

**Discussion by cattapan** [ttscoff/bunch-docs#60] [General]
I've just downloaded and started using Bunch yesterday. I'm really excited exploring it, but there's something which I feel is super important: Being able to control whether URLs are opened as separate browser instance or as a tab. Majority of my setups would include a lot of urls, but I'd like to open them in "logical groups". I.e: related stuff within the same instance of a browser, and in another instance of the browser stack overflow for instance.

Does the current version (1.3.6) support it?
I tried to just invoke multiple "Safari" applications instances, passing the urls, but in the end all opened in the same instance.

**Comment by ttscoff** [comment]
You can just send keystrokes to apps to open new windows. Launching an app twice does not open new windows, whether with Bunch or otherwise, and it would be terrible if Bunch actually started loading multiple instances of apps.

What I would recommend is just sending Command-N to the apps in question:

```
Safari
- {@n}
- http://urltopen

Terminal
- [first command\\n]
- {@n}
- [second command\\n]
```

**Comment by cattapan** [comment]


**Reply by ttscoff** [reply]
Why don't you run the terminal commands from within the Bunch?

**Comment by ttscoff** [comment]
Added a [hacky but effective solution](https://github.com/ttscoff/bunch/discussions/145#discussioncomment-1364770) in the conversation on #145.
